src/main/python/covata/delta/apiclient.py

```diff
@@ -270,7 +267,7 @@ def get_secret_content(self, requestor_id, secret_id):
         :param str requestor_id: the authenticating identity id
         :param str secret_id: the secret id to be retrieved
         :return: the retrieved secret
-        :rtype: bytes
+        :rtype: dict[str, str]
         """
```

Comment:
Shouldn't this be a string? Maybe `response.text` is more appropriate for this method